workshops/backjack/README.md

```diff
@@ -0,0 +1,1528 @@
+ 
+ We then welcome the player to the game followed by a new line using the ` \n ` (newline character)  and ask if they should write their name.
+ ```c++ 
+   cout << "\n Welcome to the game.\n Enter your Name: \n";       /*instruct the player to write their name */
+
+ ```
+ What I call the **definition section** is where we will define every function declared, example we have the ` start_the_Game() ` function declared, we then going to write the definition and inside the body.
+
+ 
+ ```c++
+/* definition section: this section will be used to define all the declared function*/
+
+void start_The_Game() {
+
+   /* definition goes here */ 
+    
+}
+
+```
+ Now that you have everything set up, let’s start some hacking!!
+ 
+ ![a cat gif typing a computer fast](https://cloud-c5sjja2wx.vercel.app/0giphy.gif)
+ 
+ ## Getting started with the main function
+ 
+
+ Now inside the **declaration section**, declare a string variable ` player1 `under the integer variable which will be used to stored the first player's name: 
+ 
+ ```c++       
+ 
+string player1;          		      /* this how it should look */
+
+ ```
+Inside the main function, we have asked the player for their name, let's then take their name using the ` getline() ` function which will take the whole line as the input, as we want the player's full name.
+
+Inside  getline function we want to get the input and stored it inside ` player1 `, so we will use  ` cin `to get input and ` player1 ` together as ` getline(cin,player1) ` : 
+
+ ```c++     
+ 
+   getline(cin,player1);                                             /* this should be written under the last line inside the main function */
+
+```
+
+Below getline() function call ` start_The_Game() ` function that will be used to start the game. 
+
+ ```c++
+
+  start_The_Game();                                                  /* will be used to start the game after the player wrote their name */
+
+```
+Note:  it important that if you *declare* a function, you should then *give Definition to* and *Call* that function.
+
+Right now ` start_The Game ` function cannot perform a task, so let give it a definition.
+
+
+## Starting the game
+
+Let now work inside the ` star_The_Game() ` function, inside this function, lets display again the introduction of the game but this time including the player's name :
+
+```C++
+/* Definition section: this section will be used to define all declared function*/
+
+void start_The_Game() {  
+   cout << "\n           Blackjack Game      \n";					 /* again print out the title to the game */
+   cout << "\n++++++++++++++++++++++++++++++++\n";					 /* print out a line using plus sign */
+   cout << "\n Welcome to the game.\n";							 /* print a welcome message */
+   cout << "\n Player of this round is: " << player1 << "\n\n"; 			 /* print the name of player */
+  }
```

Comment:
This `}` should be at an indent level 0

Reply:
@sampoder when it come to if statement I will have to use tab right?, so it readeable